src/dbus_p.h

```diff
@@ -29,6 +29,7 @@
 #include "messagepart.h"
 #include "group.h"
 
+#define COMM_HISTORY_SERVICE_NAME  QLatin1String("com.nokia.commhistory")
```

Comment:
Perhaps could be _PREFIX or something as there is no service name with this exact name.

Reply:
That's right, I've added a _PREFIX to the macro name.